### DISCUSSION TITLE
Not able to install brew on Apple M1 Pro Ventura 13.6

### DISCUSSION THREAD

**Discussion by chandra-verana** [Homebrew/discussions#4814] [Getting started] [answered]
### Output of `brew config`

```shell
% brew config
zsh: command not found: brew
```

### Output of `brew doctor`

```shell
brew doctor
zsh: command not found: brew
```

### Description of issue

This is a new laptop and i am trying to install brew. Git is preinstalled with xcode. `git version 2.39.3 (Apple Git-145)`. Complete error ```% /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
==> Checking for `sudo` access (which may request your password)...
Password:
Password:
==> This script will install:
/opt/homebrew/bin/brew
/opt/homebrew/share/doc/homebrew
/opt/homebrew/share/man/man1/brew.1
/opt/homebrew/share/zsh/site-functions/_brew
/opt/homebrew/etc/bash_completion.d/brew
/opt/homebrew

Press RETURN/ENTER to continue or any other key to abort:
==> /usr/bin/sudo /usr/sbin/chown -R username:admin /opt/homebrew
Password:
==> Downloading and installing Homebrew...
remote: Enumerating objects: 247959, done.
remote: Counting objects: 100% (25366/25366), done.
remote: Compressing objects: 100% (777/777), done.
remote: Total 247959 (delta 24720), reused 24813 (delta 24580), pack-reused 222593
Receiving objects: 100% (247959/247959), 72.38 MiB | 18.25 MiB/s, done.
fatal: could not open '.git/objects/pack/tmp_pack_KxEXio' for reading: Operation not permitted
fatal: fetch-pack: invalid index-pack output
Failed during: /usr/bin/git fetch --force origin```

Tried this ```/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/uninstall.sh)"
git config --global core.compression 0
git config --global http.postBuffer 1048576000
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"``` but got the same error

**Comment by SMillerDev** [comment]
It seems like there is some application on the laptop that prevents git from checking out and reading the Homebrew repository correctly.

**Reply by chandra-verana** [reply]
Yeah, it worked after providing full disk access in system settings to the terminal app where i am running the commands.

**Comment by chandra-verana** [comment, marked as answer]
For all the latest MAC OS versions, open system settings. In privacy window, provide full disk access to terminal/iterm app where you are running the commands. After updating the settings, the above error is resolved. Thanks!

**Reply by A-Jain02** [reply]
in the privacy->full disk access window , terminal option isn't available. Is it because I have installed VS code? Coz I can see VS code as option.
